pxr/imaging/hgiInterop/CMakeLists.txt

```diff
@@ -14,14 +14,19 @@ set(optionalPrivateHeaders "")
 # Destination of hgiInterop is always OpenGL, so always include garch
 list(APPEND optionalLibraries garch)
 
-if (PXR_ENABLE_METAL_SUPPORT)
+if (APPLEIOS)
```

Comment:
should this clause be on METAL support, and should it be within the following check?